python/lsst/cp/verify/mergeResults.py

```diff
@@ -116,6 +118,11 @@ class CpVerifyExpMergeConfig(pipeBase.PipelineTaskConfig,
         doc="Are there results tables to merge?",
         default=False,
     )
+    doDropStats = pexConfig.Field(
+        dtype=bool,
+        doc="Drop deprecated stats files?",
+        default=False,
```

Comment:
If these are deprecated, why is the default not True?  Also, is there a ticket to remove any deprecated files (if so it should be tagged here with a TODO; if not it should be filed and then tagged here with a TODO.)

Reply:
They're not formally deprecated, and this was a quick-fix to avoid the problem.  
The problem:  When I updated `cp_verify` to get the `analysis_tools` development unblocked, I was more explicit about which tasks have filter dimensions and which don't.  This changed the `flatStats` dimensions to include `physical_filter`, matching the `flatResults` dataset, but causing conflicts with the existing definition in `/repo/embargo`.  Since the `flatStats` never gest used outside of `cp_verify`, but the `flatResults` do (they're the input to `analysis_tools`), having this dataset get dropped avoided the butler problem.
The correct path forward is to RFC the `*Stats` datasets to be deprecated and removed, and I don't think that will have any objections, but having this shortcut is helpful until I have the time for the proper RFC deprecation.

Reply:
Okay, then I would suggest updating the doc string here.  Also, I'm not sure where this is even set to `True`!

Reply:
I think the pipeline that had this set to true was rebased out of existence due to RFC-1013.  I'll clarify the doc string.